packages/swift/Sources/Generated.swift

```diff
@@ -207,3 +207,137 @@ public protocol PulseWidgetEventDelegate: WidgetEventDelegate {
 public extension PulseWidgetEventDelegate {
     func widgetEvent(_: PulseWidgetEvent.OverdraftWarningCtaTransferFunds) {}
 }
+
+/** Dispatchers **/
+
+protocol Dispatcher {
+    func dispatch(_: URL)
+}
+
+extension Dispatcher {
+    func extractMetadata(_ url: URL) -> Data? {
+        return URLComponents(url: url, resolvingAgainstBaseURL: false)?
+            .queryItems?.first(where: { item in item.name == "metadata" })?
+            .value?.data(using: .utf8)
+    }
+
+    func decode<T>(_ typ: T.Type, _ data: Data) throws -> T where T: Decodable {
+        let decoder = JSONDecoder()
+        decoder.keyDecodingStrategy = .convertFromSnakeCase
```

Comment:
is `.convertFromSnakeCase` built in or custom? I don't see it defined anywhere?

Reply:
It comes from the standard library, https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy. Swift lets you use enums without fully qualifying their names. So `.convertFromSnakeCase` can be used in place of `JSONDecoder.KeyDecodingStrategy.convertFromSnakeCase` when setting the value of a variable typed as `JSONDecoder.KeyDecodingStrategy` (such as `keyDecodingStrategy`). Neat little syntax trick.